include/opendht/default_types.h

```diff
@@ -253,7 +253,7 @@ class OPENDHT_PUBLIC IpServiceAnnouncement : public Value::Serializable<IpServic
     }
 
     virtual const ValueType& getType() const {
-        return TYPE;
```

Comment:
why this change ? It builds fine for us on VS2022

Reply:
I guess that you are using the static `opendht` library to link with the tool's executables. If you link the dynamic library you will get the error here and in other similar places.